source/modulo_components/modulo_components/component_interface.py

```diff
@@ -467,15 +468,16 @@ def callback_wrapper(request, response, cb):
         try:
             parsed_service_name = parse_topic_name(service_name)
             if not parsed_service_name:
-                raise AddServiceError(f"Parsed service name is empty.")
+                raise AddSignalError(f"The parsed signal name for service {service_name} is empty. Provide a "
```

Comment:
Another copy error, should be `AddServiceError` and `f"The parsed service name ... "`